### DISCUSSION TITLE
Unable to find main class (Win)

### DISCUSSION THREAD

**Discussion by PiScOlAtOr** [ho-dev/HattrickOrganizer#991] [Q&A]
Hi All,

I've been using HO 3.0 now that I started playing again and realized there was a 4.1 recently so wanted to upgrade.
I have JRE 8 and this is a work laptop so can't really install new software like JRE 14 for v4.1.
I downloaded the portable windows version with JRE to help solve this but I get a "can't find main class" when trying to start it. running it with java -jar HO.jar tells me that main was created with a new version, assume here is the JRE14.

Does anyone know what I'm missing here ? Thanks!

**Comment by akasolace** [comment]
You should not launch it manually. Have you try to launch the executable directly: the .exe ?

**Reply by PiScOlAtOr** [reply]
Hi, thanks for the reply. Yes I try launching it manually using the HO.exe, I get the following
![image](https://user-images.githubusercontent.com/81863095/113901567-e4084c80-97c6-11eb-93ff-c3e8c55db561.png)
it is only when I try to run the HO.jar from the cmd window that I see that it talked about not being the latest version as below
![image](https://user-images.githubusercontent.com/81863095/113901883-347faa00-97c7-11eb-90dc-4d2743ec00f4.png)

**Comment by akasolace** [comment]
I just downloaded HO-5.0.3557.0-portable-win-DEV-JRE.exe  and it is working for me ....

**Reply by akasolace** [reply]
also works with HO-4.1.3310.2-portable-win-JRE.exe

**Comment by akasolace** [comment]
problem is I have the JDK installed on my PC so I am not sure if it used as a fallback ....

**Comment by akasolace** [comment]
still I am surprised no one else reported this as I know the portable version with JRE is being used by plenty of users .....

**Comment by PiScOlAtOr** [comment]
Hi, today I decide to give it another try  and to my surprise I can actually open it now, not sure what has changed, just coming back from leave :)
Unfortunately I get stuck on another step, when initiating my connection for the first time, the URL to click is empty to can't register the CHPP, any idea of why that might be? using the latest portable version with JRE
Regards,

**Reply by akasolace** [reply]
If working on a professional laptop maybe you are behind a firewall. Can you access Hattrick website from a browser ?
